src/constants/hardwareTypes.ts

```diff
@@ -1,41 +1,83 @@
-// GAS RM on cpu STM32L15x - no longer in production, over 10,000 were produced.
+/**
+ * GAS RM on cpu STM32L15x - no longer in production, over 10,000 were produced.
+ */
 export const GAZM3 = 1;
 
-// GAS RM on cpu STM32L05x - no longer in production, several thousand produced.
+/**
+ * GAS RM on cpu STM32L05x - no longer in production, several thousand produced.
+ */
 export const GAZM0 = 2;
 
-// GAS RM on cpu STM32L05x - test batch
+/**
+ * GAS RM on cpu STM32L05x - test batch.
+ */
 export const GAZM0NEW = 3;
 
-// RM for novator
+/**
+ * RM for novator.
+ *
+ * 2 channels
+ */
 export const NOVATOR = 4;
 
-// RM with 2 ports, no longer in production
+/**
+ * RM with 2 ports, no longer in production.
+ *
+ * 2 channels
```

Comment:
important to know amount of channels of hardware. Rework this or new constants?

Reply:
mayb leter, let's get more info first